.github/workflows/continous-deployment.yml

```
@@ -38,4 +38,4 @@ jobs:
           echo "production_url=$url" >> $GITHUB_OUTPUT
 
       - name: Show Production URL
-        run: echo "Production URL: ${{ steps.deploy.outputs.production_url }}"
\ No newline at end of file
+        run: echo '"Production URL: ${{ steps.deploy.outputs.production_url }}"'
```

actionlint (1.7.11)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)